docs/How-To-Guides/how-to-create-play.md

```diff
@@ -90,6 +92,8 @@ npx create-react-play@latest -c <the_play_id>
   npm run start
   ```
 
+```
```

Comment:
These backticks on lines 95 and 126 need to be removed. It's not getting rendered as expected.
Preview - https://deploy-preview-66--react-play-docs.netlify.app/How-To-Guides/how-to-create-play

Reply:
@supminn  Done, check it once now.